packages/blade/src/components/Amount/_decisions/decisions.md

```diff
@@ -0,0 +1,88 @@
+# Amount <!-- omit in toc -->
+
+A amount component is a UI element that displays and formal various currency values in a specified format.
+
+<img  src="./amount-thumbnail.png"  width="380" />
+
+## Design
+
+- [Figma - Amount](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=28012%3A580578&t=3peAz8A2n2Gw4WMl-1)
+
+## API
+
+| Prop | Type | Default | Description | Required |
```

Comment:
Table formatting is broken, please validate in markdown preview.

Reply:
Fixed.